src/useSanityShopifyProducts.ts

```diff
@@ -12,14 +12,15 @@ function getQuery(products: ProductWithFragment[]): string {
   // @TODO: replace with final ProductProviderFragment
   return `
   query getProducts(
+    $country: CountryCode
     $numProductMetafields: Int!
     $numProductVariants: Int!
     $numProductMedia: Int!
     $numProductVariantMetafields: Int!
     $numProductVariantSellingPlanAllocations: Int!
     $numProductSellingPlanGroups: Int!
     $numProductSellingPlans: Int!
-  ) {
+  ) @inContext(country: $country) {
```

Comment:
Following up on the comment in `getShopifyVariables`, perhaps this portion could be `${country ? "@inContext(country: $country)" : ""}`?